scripts/error_code_generator_defs/result_codes.json

```diff
@@ -991,6 +1007,85 @@
               {
                 "name": "ADUC_ERC_UTILITIES_ROOTKEYPKG_PARSE_INVALID_EXPONENT",
                 "value": 28
+              },
+              {
+                "name": "ADUC_ERC_UTILITIES_ROOTKEYPKG_DOWNLOAD_URL_PARSE",
+                "value": 29
+              },
+              {
+                "name": "ADUC_ERC_UTILITIES_ROOTKEYPKG_DOWNLOAD_HTTP_URL_PATH_PARSE",
+                "value": 30
+              },
+              {
+                "name": "ADUC_ERC_UTILITIES_ROOTKEYPKG_DOWNLOAD_URL_BAD_PATH",
+                "value": 31
+              },
+              {
+                "name": "ADUC_ERC_UTILITIES_ROOTKEYPKG_DOWNLOAD_BADARG",
+                "value": 32
+              },
+              {
+                "name": "ADUC_ERC_UTILITIES_ROOTKEYPKG_DOWNLOAD_MKDIR_DWNLD_FOLDER",
+                "value": 33
+              },
+              {
+                "name": "ADUC_ERC_UTILITIES_ROOTKEYPKG_DOWNLOAD_FAILED",
+                "value": 34
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_INIT",
+                "value": 35
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_SETOPT_PROTOCOLS",
+                "value": 36
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_SETOPT_URL",
+                "value": 37
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_SETOPT_TIMEOUT",
+                "value": 38
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_SETOPT_WRITEFUNCTION",
+                "value": 39
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_SETOPT_WRITEDATA",
+                "value": 40
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_PERFORM_DOWNLOAD",
+                "value": 41
+              },
+              {
+                "name": "ADUC_ERC_ROOTKEYPKG_DOWNLOADER_CURL_FILEWRITE",
+                "value": 42
+              }
+            ]
+          },
+          {
+            "code": 6,
+            "doc_string": "indicates errors from url utility.",
+            "name": "ADUC_COMPONENT_URL_UTIL",
+            "results": [
+              {
+                "name": "ADUC_ERC_UTILITIES_URL_CREATE",
```

Comment:
0 isn't a good idea for a result code. Sorry didn't catch this earlier just figured I'd check

If we use 0 then the result code technically maps to the same value as the component code which can be confusing. 

It's ok if you want to keep it, I just went through and searched, and it works fine but something to keep in mind.

Reply:
Good catch.  Changed to start at 1